geopandas/plotting.py

```diff
@@ -188,34 +189,17 @@ def plot_point_collection(ax, geoms, values=None, color=None,
     x = geoms.x.values
     y = geoms.y.values
 
+    # scatterplot uses different way to specify size as default plot markersize
+    if markersize:
+        markersize = markersize ** 2
+
```

Comment:
I think that default is set though so that if a user plots points with either `plot` or `scatter` without specifying styling, the appearance will be the same. If a user passes in a size to `scatter`, it doesn't then square that. So I'd personally lean towards taking `markersize` as the user defines it, and noting that the specified `markersize` follows the `scatter` convention, and gives an area of the point (or at least something proportional to area), rather than diameter.

Reply:
The question is then maybe: if a user would add an additional point(s) to the same plot just uskng matplotlib, would he/she rather use plot or scatter? Don't really know.
Either way, you are correct it should be better documented

Reply:
I'd personally be likelier to use scatter in such a situations, but I have no idea how representative I am.

Reply:
Yes, you are probably right. Because a typical use case where you care about the actual markersize (apart from tweaking for aesthetics), is when you want varying sizes based on a certain column, on for this the scatter behaviour is a better default (size is better representation of a value than the diameter).

Changed this back.